tests/release/pipelines/multiarch_advisories.go

```diff
@@ -173,12 +189,32 @@ var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for multi arch test f
 					if err != nil {
 						return err
 					}
-					GinkgoWriter.Println("Release CR: ", releaseCR.Name)
-					if !releaseCR.IsReleased() {
-						return fmt.Errorf("release %s/%s is not marked as finished yet", releaseCR.GetNamespace(), releaseCR.GetName())
+					GinkgoWriter.Println("releaseCR: %s", releaseCR.Name)
```

Comment:
This block of code is repeated in a lot of tests. Maybe extracting to to a lib function would be a better idea then duplicating it so many times

Reply:
Thanks for the advice. Created a function `CheckReleaseStatus` in releaseLib.go.